### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by xbrianh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,44 @@
...
+and [bcftools](https://github.com/samtools/bcftools) available through pypi packaging. These tools have been lightly
+modified to allow merges on VCF streams without an index.
 
-# installation
+# Installation
 ```
 pip install xsamtools
 ```
-A C toolchain is required to install xsamtools.
+Installation requires a C toolchain.
```
Comment (DailyDreaming) [inline README.md:10]: I would include the dependencies needed for an Ubuntu (any Debian?) system:

```sudo apt install libcurl4-openssl-dev libbz2-dev liblzma-dev```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,44 @@
...
+
+# Usage
+
+After succesful installation, the following executables are available:
```
Comment (DailyDreaming) [inline README.md:14]: What happens if `htsfile` is already installed on the system, is it overwritten?

File: README.md [hunk 3]
```diff
@@ -1,10 +1,44 @@
...
+```
+These streams appear as either readable or writable files on the filesystem. Such objects are not seekable.
+
+A utility method is also provided to merge VCFs from GS objects using the streams discussed above:
+```
+from xsamtools import vcf
+
+vcf.combine("src-bucket-name", ["first-src-vcf-key", "second-src-vcf-key"], "dst-bucket-name", "dst-vcf-key")
+```
+There is no formal limit on the number of VCF keys. Care should be taken that the VCF objects provided are aligned by
```
Comment (DailyDreaming) [inline README.md:43]: Would it make sense to have an issue created for this to automatically be detected by the program?

File: README.md (hunk 2 above)
Comment (xbrianh) [reply README.md:14]: If you're installing into a virtual environment the binaries will end up there. For system installs I suppose it's a possibility.

File: README.md (hunk 3 above)
Comment (xbrianh) [reply README.md:43]: As far as I'm aware, it's not possible or feasible to detect non-aligned VCF streams. Users need to know if their VCFs are aligned.

File: README.md (hunk 1 above)
Comment (xbrianh) [reply README.md:10]: will add

**Pull request merged**